### ISSUE TITLE
As an animator I want an EMFX anim graph comment node

### ISSUE THREAD

**Issue by AMZN-alexpete** [o3de/o3de#13318]
**Is your feature request related to a problem? Please describe.**
I often want to leave comments in anim graphs especially to explain complexity or important aspects of the design

**Describe the solution you'd like**
I would like a comment node that I can add to the anim graph and enter text into.  The node should be resizable to fit the text.
The node should not be present in the final game-asset anim graph.

**Describe alternatives you've considered**
Long headers in group nodes could be used for simple short comments.

**Additional context**
n/a

**Comment by hultonha** [comment]
Agreed this would be a useful feature but not sure exactly when we will have capacity to work on it.

Tagging @o3de/sig-ui-ux for visibility if they have time to create a mock of how they would want such a feature to be implemented.

**Comment by derikson-dev** [comment]
This issue still needs to be fixed.
